frontend/src/routes/_dashboard/create-project.lazy.tsx

```diff
@@ -28,11 +35,50 @@ function AuthedView() {
 }
 
 export function CreateProject() {
+  const navigate = useNavigate();
+
+  const { isPending, mutate, data } = useMutation({
+    mutationFn: async (input: RequestInput<"post", "/api/projects/">) => {
+      await apiClient.GET("/api/csrf/");
+      const csrfToken = getCookie("csrftoken");
+      const { error, data } = await apiClient.POST("/api/projects/", {
+        headers: {
+          "X-CSRFToken": csrfToken
+        },
+        body: input
+      });
+
+      if (error) return error;
+      if (data) {
+        navigate({ to: "/" });
```

Comment:
This should redirect to the project details page. Since this page doesn't exist yet, you can create a sample page at `/project/{slug}` and redirect to it using the slug of the project returned by the API

Reply:
yes for now we will direct on the /